app/controllers/pages_controller.rb

```diff
@@ -9,16 +9,17 @@ def about
 
   def feed
     @user = current_user
-    @users = User.all
-    @events = Event.all
+    @users = User.all.sample(10)
+    @user = User.all
```

Comment:
What is @user = User.all giving access to?

Reply:
maybe needs to be find_by id ?